### ISSUE TITLE
Geospatial indexes

### ISSUE THREAD

**Issue by nikolayg** [matthaywardwebdesign/aus-search#1]
First of all, this is a great tool - thanks for building it.

I was looking at `config/indexes.json` and I couldn't see any [geospatial indexes](https://docs.mongodb.com/manual/applications/geospatial-indexes/#geospatial-indexes). Any guidance on how to quickly add geo indexes would be appreciated!

**Comment by matthaywardwebdesign** [comment]
@nikolayg I'm guessing you were planning on querying the data using MongoDB then? :)

**Comment by nikolayg** [comment]
Ah yes, in Mongo. I tweaked your code a little bit and managed to get `2dsphere` indexes.

On another note - there's an issue when you try to populate a remote mongo server over the network. The scripts read the files and populate the buffers much faster than the DB insertion. This results in out-of-memory exceptions. Even 4G of `--max-old-space-size` didn't help. I ended up running the scripts with a local Mongo, and then imported the dumps in the server.
